pymobiledevice3/cli/developer.py

```diff
@@ -97,6 +99,21 @@ def applist(lockdown, color):
         print_json(apps, colored=color)
 
 
+@dvt.command('signal', cls=Command)
+@click.argument('pid', type=click.INT)
+@click.argument('sig', type=click.INT, required=False)
+@click.option('-s', '--signal-name', type=click.Choice([s.name for s in signal.Signals]))
+def send_signal(lockdown, pid, sig, signal_name):
```

Comment:
rename to `signal`

Reply:
If I rename to signal I need to do:
Import signal as [something] or else I have a a collision